apps/qmsched/lib.js

```diff
@@ -3,23 +3,36 @@
  * @param {int} mode Quiet Mode
  */
 function switchTheme(mode) {
-  if (!!mode === g.theme.dark) return; // nothing to do
   let s = require("Storage").readJSON("setting.json", 1) || {};
+  print("Theme is ", s.theme);
```

Comment:
Forgotten debug statement?

Reply:
Yes, exactly.

Reply:
Interesting, I pushed a new commit to remove this, but it does not (yet?) show up here. But it does here: https://github.com/halemmerich/BangleApps/commit/d60b2da468f280b0fbbb822a6520711eb81a2719